public-front/docker/app/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM composer AS composer
 
 COPY public-front /app
 
-RUN composer install --prefer-dist --no-suggest --no-interaction --no-scripts --optimize-autoloader
+RUN composer install --prefer-dist --no-suggest --no-interaction --no-scripts --optimize-autoloader --ignore-platform-reqs
```

Comment:
Is this potentially a bad idea, as it means that composer won't check that the container is running the correct PHP version? I understand why we'd want this on containers which won't necessarily have the right PHP version where we want to mock APIs (e.g. test containers), but not sure why we need to do this on the "real" docker container.

Reply:
Ah, can now see this is happening in the composer sub-image (not sure what the correct term is), so should be OK.

Reply:
Just talking to myself now...

Reply:
ah - just spotted this blog. perhaps i could just ignore the specific extension at this point. https://php.watch/articles/composer-ignore-platform-req

Reply:
the issue is to do with this: 

``` sh 
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - laminas/laminas-i18n is locked to version 2.11.0 and an update of this package was not requested.
    - laminas/laminas-i18n 2.11.0 requires ext-intl * -> it is missing from your system. Install or enable PHP's intl extension.

```
so either we ignore this or add the extension.

Reply:
Right, I think for the reasons I mentioned, this is OK, as the required deps will be on the main image.

Reply:
Added the extension. should be ok now.